src/api-request.js

```diff
@@ -1,42 +1,31 @@
 let apiRequest = {
-  userInfo: getUserData(),
-  sleepInfo: getSleepData(),
-  hydrationInfo: getHydrationData(),
-  activityInfo: getActivityData()
-};
-function getUserData() {
-  return fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
+getUserData() {
+    return fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
     .then(response => response.json())
     .then(data => data.userData)
     .catch(error => console.log(error));
+},
```

Comment:
That's an interesting way to organize this data inside the object literal! I learned something new!